include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -180,6 +180,9 @@ make_iter_mode(const _Iterator& __it) -> decltype(iter_mode<outMode>()(__it))
 
 // set of class templates to name kernels
 
+template <typename... _Name>
```

Comment:
Please remove since parallel_for doesn't require to have a name. It should work without this `class` declaration

Reply:
Done